kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/ExecWebSocketListener.java

```diff
@@ -349,7 +349,6 @@ private static InputStream inputStreamOrPipe(InputStream stream, PipedOutputStre
             return stream;
         } else if (out != null) {
             PipedInputStream pis = bufferSize == null ? new PipedInputStream() : new PipedInputStream(bufferSize.intValue());
-            toClose.add(out);
```

Comment:
+1: You should never ever be your responsibility to close a stream that was handed out to you.